web/template/layout/page.gohtml

```diff
@@ -16,7 +16,7 @@
       <link rel="apple-touch-icon" sizes="152x152" href="{{ prefix "/assets/images/govuk-apple-touch-icon-152x152.png" }}">
       <link rel="apple-touch-icon" href="{{ prefix "/assets/images/govuk-apple-touch-icon.png" }}">
 
-      <link href="{{ prefix "/stylesheets/all.css" }}" rel="stylesheet">
+      <link href="{{ prefix "/stylesheets/all.css?v1" }}" rel="stylesheet">
```

Comment:
I've added https://opgtransform.atlassian.net/browse/VEGA-1276 to look at doing this properly, this is just the simplest thing to avoid making this PR any bigger.